project/Dependencies.scala

```diff
@@ -120,4 +120,13 @@ object Dependencies {
     )
   )
 
+  val GooglePubSub = Seq(
+    libraryDependencies ++= Seq(
+      "com.typesafe.akka"     %% "akka-http"     % AkkaHttpVersion,
+      "com.typesafe.akka" %% "akka-http-spray-json" % AkkaHttpVersion,
+      "org.mockito"   % "mockito-core"     % "2.3.7"    % Test, // MIT
+      "com.github.tomakehurst" % "wiremock"      % "2.3.1" % Test //ApacheV2
```

Comment:
Im not sure what you mean. I would have thought standalone is not for use in tests like this.
I've upgraded to 2.5.1